test/Interop/Cxx/value-witness-table/Inputs/basic.h

```diff
@@ -0,0 +1,36 @@
+#ifndef TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
+#define TEST_INTEROP_CXX_DESTRUCTOR_INPUTS_BASIC_H
+
+struct HasUserProvidedDestructor {
+  int *value;
+  ~HasUserProvidedDestructor() { *value = 42; }
+};
+
+struct EmptyDestructor {
+  HasUserProvidedDestructor box;
+  ~EmptyDestructor() { /* empty */
+  }
+};
+
+struct HasNonTrivialImplicitDestructor {
+  HasUserProvidedDestructor box;
+};
+
+struct HasNonTrivialDefaultDestructor {
```

Comment:
```suggestion
struct HasNonTrivialDefaultedDestructor {
```
I think this is the phrase used in the C++ spec.

Reply:
Good call. Done.